webpack.config.base.js

```diff
@@ -37,6 +40,27 @@ module.exports = {
             static: path.resolve(__dirname, './src/assets'),
         },
     },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
```

Comment:
do we need to do this on the renderer as well? if not, we probably want to have this only in the config for the main process.